src/SettingsServiceProvider.php

```diff
@@ -36,9 +37,15 @@ public function boot()
         $this->setupRoutes($this->app->router);
 
         // only use the Settings package if the Settings table is present in the database
-        if (!\App::runningInConsole() && count(Schema::getColumnListing('settings'))) {
-            // get all settings from the database
-            $settings = Setting::all();
+        $tableExists = Cache::remember('backpack_settings_table_exists', config('backpack.base.cache_time', 60), function () {
+            return count(Schema::getColumnListing('settings'));
+        });
+
+        if (!\App::runningInConsole() && $tableExists) {
+            // get all settings from the database if they're not in the database.
+            $settings = Cache::remember('backpack_settings_cache', config('backpack.base.cache_time', 60), function () {
```

Comment:
I have the same concern here - related to ```config('backpack.base.cache_time', 60)```

Reply:
I've adjusted this and updated the PR to merge with current code.  I couldn't find a default cache time in Laravel and have added this setting in the backpack.settings file.

It's also been updated with the suggested code from here: https://github.com/Laravel-Backpack/Settings/issues/90#issuecomment-565978500